webpack.config.js

```diff
@@ -457,17 +527,17 @@ module.exports = (env, argv) => {
         // configuration for the webpack-dev-server
         devServer: {
             // serve unwebpacked assets from webapp.
-            contentBase: './webapp',
+            contentBase: [
+                './webapp',
+            ],
 
             // Only output errors, warnings, or new compilations.
             // This hides the massive list of modules.
             stats: 'minimal',
-
-            // hot module replacement doesn't work (I think we'd need react-hot-reload?)
-            // so webpack-dev-server reloads the page on every update which is quite
-            // tedious in Riot since that can take a while.
-            hot: false,
-            inline: false,
+            // hot: false,
```

Comment:
Leaving it commented here, webpack-dev-server is taking care of that, but it's not a bad idea to just remember we have it.